### ISSUE TITLE
Lack of Documentation

### ISSUE THREAD

**Issue by mateeyow** [bootflat/bootflat.github.io#63]
Might I suggest a proper documentation in using your framework just like bootstrap and foundation, I think it will be easier for us developers to use your framework when you have better documentation. Thank you!

**Comment by hitchcockwill** [comment]
Yep, came here to add an issue like this. 

Another suggestion for an easy improvement would be to not minify the HTML on the documentation page. At the very least, it would make it a lot easier to inspect the source.

**Comment by mojowill** [comment]
Guys what documentation do you want? You should be referring to the Bootstrap documentation on most things, BootFlat is simply a UI kit that alters the appearance of standard Bootstrap elements, for other BootFlat specific stuff there are examples on the website, why not simply "Inspect Element"?

:-1:

**Comment by DerDu** [comment]
@hitchcockwill If you use e.g Chrome and Inspect Element it got the little"{}" button to reformate the source code for good reading...

I would like a documentation of the missing parts not in Bootstrap but added by BootFlat e.g the footer .. thats all

**Comment by nomadcrypto** [comment]
The lack of unminifed sources and documentation for those unminifed sources makes it hard to work with. Having either 100% of unminified sources available or proper documentation for those sources is what is needed to make this usable for a real world project.

**Comment by tdeck** [comment]
+1 to this. I really like the way Bootflat looks, and I would love to start using it, but as I dig deeper I've really become frustrated by how there are no docs for anything at all. I understand that most of this stuff is the same for ordinary Bootstrap, but there are some things that seem to be Bootflat specific as mentioned above. One good example is the color theming which appears to be heavily featured on the Bootlfat website, but I can't find a simple way to change the colors - I assume the right answer isn't to just drop to raw CSS.

**Comment by mallim** [comment]
if bootflat can be structured something like bootswatch (https://bootswatch.com/), it will also be quite easy to understand in terms of the customization done to boostrap.

**Comment by les113** [comment]
Documentation? Do you really think this http://bootflat.github.io/documentation.html passes for documentation?

**Comment by rhssk** [comment]
I agree that the documentation leaves something to be desired. Maybe add source code (HTML, JS) for each of the elements in http://bootflat.github.io/documentation.html? In my opinion it should be a quick reference (https://designmodo.github.io/Flat-UI/docs/components.html is a good example). Now you have to inspect page source, filter out the extras and go hunting documentation for each of the used tools (iCheck etc).

**Comment by shayneoneill** [comment]
Seconding this. We've got a getting started that tells you to download it (kinda obvious, but ok), and doesn't tell us how to get started.

And a documentation thats a demo but has no documentation. 

Even just some indication of what files in that big ol' ball of stuff we're supposed to use would go a long way.

**Comment by nomadcrypto** [comment]
i requested this documentation several years ago before I knew what I know
now about css, html, less, etc. Now I wouldn't need it like I did then but
overall proper documentation is always nice. I do like the UI in bootflat
but now there are several alternatives which have decent documentation. An
example is adminlte - https://github.com/almasaeed2010/AdminLTE
